src/fides/data/sample_project/cookie_house/src/lib/db.ts

```diff
@@ -0,0 +1,18 @@
+import { Pool } from 'pg';
+
+const { DATABASE_HOST, DATABASE_PORT, DATABASE_USER, DATABASE_PASSWORD, DATABASE_DB } = process.env;
+if (!DATABASE_HOST) throw new Error('No DATABASE_HOST');
+if (!DATABASE_PORT) throw new Error('No DATABASE_PORT');
+if (!DATABASE_USER) throw new Error('No DATABASE_USER');
+if (!DATABASE_PASSWORD) throw new Error('No DATABASE_PASSWORD');
+if (!DATABASE_DB) throw new Error('No DATABASE_DB');
+
+const pool = new Pool({
+  host: DATABASE_HOST,
+  port: +DATABASE_PORT,
```

Comment:
Is the preceding `+` necessary?

Reply:
Yes- it's a clever way to satisfy Typescript, basically, since the `Pool` constructor expects a number but all env variables come back as strings.

The contractor did this though, personally I'd probably have cast it as `any` and did a lint ignore, but TIL you could do this